EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1699,59 +1699,59 @@ QUEST_LV_0300_20160310_001698	Move to Veyrhas Hall with Villager Duncan
 QUEST_LV_0300_20160310_001711	On the way out from the Annex Laboratory, you came across Priest Dominikas. Talk to him.
 QUEST_LV_0300_20160310_001712	Move to the Principal Corridor
 QUEST_LV_0300_20160310_001713	Priest Dominikas praises you for those who can walk through the black wall is those whom the Divine Tree allowed. He says an important ritual is about to commence and he wants to meet him at the Principal Corridor.
-QUEST_LV_0300_20160310_001714	Attend Dominikas' Ritual
-QUEST_LV_0300_20160310_001715	Dominkas said an important ritual is to be held at the Principal Corridor. Attend the ritual.
+QUEST_LV_0300_20160310_001714	Attend Dominikas' ritual
+QUEST_LV_0300_20160310_001715	Dominikas said an important ritual is to be held at the Principal Corridor. Attend the ritual.
 QUEST_LV_0300_20160310_001716	Go to the Elgos Abbey Main Building
 QUEST_LV_0300_20160310_001717	Dominikas' ritual was a ruse to accuse you of heresy. Escape to the Elgos Abbey Main Building.
-QUEST_LV_0300_20160310_001718	Remove the Vines
+QUEST_LV_0300_20160310_001718	Remove the vines
 QUEST_LV_0300_20160310_001719	Talk to Elgos Monk
 QUEST_LV_0300_20160310_001720	The monk is in need of your help. Talk to him.
 QUEST_LV_0300_20160310_001721	Elgos Monk wants to get rid of all the vines Dominikas has grown. Talk to Lucienne Winterspoon to get the info you need. 
 QUEST_LV_0300_20160310_001722	The Elgos Monk is looking for a way to remove all the vines grown by Dominikas. Talk to Lucienne Winterspoon to get the info you need. 
-QUEST_LV_0300_20160310_001723	Remove the Vines by Using Lucienne's Special Reagent
+QUEST_LV_0300_20160310_001723	Remove the vines by using Lucienne's Special Reagent
 QUEST_LV_0300_20160310_001724	Lucienne Winterspoon has developed new reagent. She combined materials in accordance with the adivice from Vaidotas. The result was a safe reagent.
 QUEST_LV_0300_20160310_001725	The vines wither away from a mere touch of Lucienne's special reagent.
 QUEST_LV_0300_20160310_001726	The Elgos Monk is in need of your help. Talk to him.
-QUEST_LV_0300_20160310_001727	Retrieve the False Scripture
-QUEST_LV_0300_20160310_001728	The monk wants retrieve the false scripture spread by Dominikas. Inspect the bookpiles in the Veyrhas Hall.
-QUEST_LV_0300_20160310_001729	You've found the false scripture from the pile. Return it to the Elgos Monk.
-QUEST_LV_0300_20160310_001730	According to the Tree of Truth sect Order, they trapped the monk in the principal building. Find the trapped monk.
+QUEST_LV_0300_20160310_001727	Retrieve the fake scripture
+QUEST_LV_0300_20160310_001728	The monk wants retrieve the fake scripture spread by Dominikas. Inspect the bookpiles in Veyrhas Hall.
+QUEST_LV_0300_20160310_001729	You've found the fake scripture from the pile. Return it to the Elgos Monk.
+QUEST_LV_0300_20160310_001730	According to the Tree of Truth sect, they trapped the monk in the principal building. Find the trapped monk.
```

Comment:
"they trapped a monk" ?

It doesn't seem like the monk that's trapped is specifically mentioned before?